contrib/internal/httputil/trace.go

```diff
@@ -17,30 +17,44 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
+// TraceConfig defines the configuration for request tracing.
+type TraceConfig struct {
+	Writer         http.ResponseWriter       // response writer
```

Comment:
This is not a "writer", it's a "response writer", which is different (think `io.Writer`)
```suggestion
	ResponseWriter         http.ResponseWriter       // response writer
```